web/src/HiddenNetworkForm.jsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+
+import {
+  Button,
+  Card,
+  CardBody,
+  Split,
+  SplitItem,
+} from "@patternfly/react-core";
+
+import Center from "./Center";
+import WifiConnectionForm from "./WifiConnectionForm";
+
+function HiddenNetworkForm({ network, expanded, onClick, onCancel, onSubmitCallback }) {
+  return (
+    <>
+      <Card className={[
+        "selection-list-item",
+        expanded && "selection-list-focused-item",
+        !expanded && "collapsed"
+      ].join(" ")}
```

Comment:
NOTE: better to use a util function to improve this and avoid _"false"_ CSS classes. 

See https://github.com/jangouts/jangouts/blob/2081d8f344b016a884b58f651a46aa69ece5070a/src/utils/common.js#L8-L18

```js
/**
 * Helper to simplify building the className prop conditionally
 *
 * @param {String} CSS classes to join
 * @returns {String} final CSS classes properly joined, exluding "false"s
 */
function classNames(...classes) {
  return classes.filter((item) => !!item).join(' ');
}
```

Reply:
Done, see https://github.com/yast/d-installer/pull/292/commits/4491ee2c0651c589f7a1e6add535549e7b825be1